src/components/page/MembersPage.tsx

```diff
@@ -25,10 +25,15 @@ export const MembersPage = async ({ id }: IMembersPageProps) => {
       <div className="flex flex-col gap-2 pb-1">
         {users.map((user, i) => (
           <Card key={i} asChild>
-            <Button className="flex gap-3 justify-start items-center">
```

Comment:
Each member card don't seem to be button, since they don't have any click event in design. So I changed it to `div`, even though it's not related to this task's scope

Reply:
Thanks for fixing it!